tests/support/globalSetup.ts

```diff
@@ -1,31 +1,105 @@
 import type { FullConfig } from "@playwright/test";
 
-import { seedPort, seedRemote } from "@tests/support/fixtures.js";
+import * as Fs from "node:fs/promises";
+import * as FsNode from "node:fs";
+import * as Path from "node:path";
+import * as Process from "./process.js";
+import { createPeerManager } from "./peerManager.js";
+import { execa } from "execa";
+import { gitOptions } from "./fixtures.js";
+import { sleep } from "@app/lib/sleep";
+import { fileURLToPath } from "url";
+import { dirname } from "path";
+
+const filename = fileURLToPath(import.meta.url);
+const dir = dirname(filename);
 
 export default async function globalSetup(_config: FullConfig): Promise<void> {
-  await assertHttpApiRunning();
-}
+  await assertRadInstalled();
 
-// Assert that the test http-api is running. If it is not running, throw an
-// error that explains how to run it.
-async function assertHttpApiRunning(): Promise<void> {
-  const notRunningMessage =
-    "The http-api server with test fixtures needs to be running.\n" +
-    "👉 You can start it with `./scripts/run-httpd-with-fixtures`\n";
-
-  let nodeId: string | undefined = undefined;
-
-  try {
-    const response = await fetch(`http://0.0.0.0:${seedPort}/api/v1`);
-    const data = await response.json();
-    nodeId = data.node.id;
-  } catch (err) {
-    console.error(err);
-    throw new Error(notRunningMessage);
-  }
+  console.log("Setting up global test environment");
+  console.log("Resetting environment");
+  const tmp = Path.resolve(dir, "..", "./tmp");
+  const projectName = "source-browsing";
+  const sourceBrowsingDir = Path.join(tmp, "repos", projectName);
+  await Fs.rm(Path.join(tmp, "peers"), { recursive: true, force: true });
```

Comment:
I'm a bit weary about deleting stuff from a user's computer without any prompts. Especially because this is run as part of the test suite, it might be unexpected. Also if we have a bug in the code it could go horribly wrong. That is why in the original script we had a `--non-interactive` flag, which we used on CI and by default the script showed the user what would be deleted and prompted for a confirmation.

I don't have a clear idea how to do it here now that we moved it out of a shell script. But might be worth to think about it a bit more.

Reply:
Yeah I concurr with you, will think on how we can get a prompt user before deletion of files

Reply:
Ok I think I found a nice way, using the node `readline` interface.

Reply:
Ok with 892fa0d we can check if we are running CI and if not, we show the user a prompt to confirm the deletion of the `repos` and `peers` folder.